src/main/java/nextstep/ladder/domain/ladder/LadderPosition.java

```diff
@@ -0,0 +1,53 @@
+package nextstep.ladder.domain.ladder;
+
+import nextstep.ladder.exception.OutOfRangeArgumentException;
+
+import java.util.Objects;
+
+public class LadderPosition {
+    private static final int ZERO = 0;
+
+    private int position;
+
+    private LadderPosition(int position) {
+        validate(position);
+        this.position = position;
+    }
+
+    private void validate(int position) {
+        if (position < ZERO) {
+            throw OutOfRangeArgumentException.greaterThan(ZERO, position);
+        }
+    }
+
+    public static LadderPosition from(int position) {
+        return new LadderPosition(position);
+    }
+
+    public LadderPosition moveLeft() {
+        --position;
+        return this;
+    }
+
+    public LadderPosition moveRight() {
+        ++position;
+        return this;
+    }
```

Comment:
```suggestion
    public LadderPosition moveLeft() {
        return new LadderPoisition(position - 1);
    }

    public LadderPosition moveRight() {
        return new LadderPoisition(position + 1);
    }
```
이렇게 새로운 객체를 만들어서 내보내줘도 괜찮을 것 같아요!

Reply:
네네, ㅎㅎ 객체의 값을 바꿔 리턴할까, 새로운 객체를 생성해서 리턴할까 하다가, 객체 생성 비용을 줄이는 방식을 채택해봤어요. 
현재 규모에선 큰 의미가 있진 않으니 새 객체를 생성하는 방식이 position 값이 음수가 되는 버그도 잡아낼 수 있고, [불변 객체](https://mangkyu.tistory.com/131)가 주는 이점을 누리는게 더 좋겠네요